<h2>RNA-seq of muskuloskeletal tissues from <i>Mus musculus</i></h2>

<h3>Tissues</h3>
<ul>
    <li>Articular cartilage</li>
    <li>Growth plate</li>
    <li>Bone</li>
    <li>Tendon</li>
    <li>Muscle</li>
    
</ul>
<h3>Articular cartilage</h3>
<p>
    This dataset was published by <a href="https://doi.org/10.1073/pnas.201915211" target="_blank">Haseeb et al., 2021</a> 
    and raw data can be found in the BioProject <a href="https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA646069" target="_blank">PRJNA646069</a>.<br>
    Tissue source: LCM articular cartilage.<br>
    Age: 12 weeks old.<br>
    Sex: Both.<br>
    Layout: PAIRED.<br>
    Reads length: 100 bp.<br>
    Experimental procedure: Knee joints were harvested from control and mutant mice and were snap frozen in OCT compound. Frozen sections were cut and then processed for laser-caputure microdissection. Articular and growth plate cartilage area were removed using laser-capture mirodissection, flash frozen on dry ice, and RNA was harvested using Qiagen Kit RNeasy Plus Micro Kit. SMARTer Stranded Total RNA-seq v2 Pico Input Mammalian kit (Takara Bio) was used for the construction of sequencing libraries.
    RNA libraries were prepared for sequencing using standard Illumina protocols. Sequencing was performed with Illumina NovaSeq 6000 and normalized to TPM.

</p>
<h3>Growth plate</h3>
<p>
    <b>Epiphysis</b><br>
    <br>
    This dataset was published by <a href="https://doi.org/10.1016/j.bone.2022.116391" target="_blank">Taylor et al., 2022</a> 
    and raw data can be found in the BioProject <a href="https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA793842" target="_blank">PRJNA793842</a>.<br>
    Tissue source: Immature epiphyseal chondrocytes isolated by collagenase digestion overnight.<br>
    Age: 5 days old.<br> 
    Sex: Males.<br>
    Layout: PAIRED.<br>
    Reads length: 100 bp.<br>
    Experimental procedure: Primary epiphyseal chondrocytes were harvested from five-day-old male wildtype (n=3) mice. Chondrocytes were digested for one hour in collagenase II in DMEM culture medium. Cells were isolated, washed in ice-cold PBS three times, then resuspended in Trizol for RNA extraction. Total RNA was extracted using phenol-chloroform.
    TruSeq RNA method was used for analysis of polyadenylated mRNAs that were selected using oligo dT magnetic beads. TruSeq Kits were used for indexing to permit multiplex sample loading on the flow cells of an Illumina HiSeq 2000 sequencer and normalized to TPM.<br>
    <br>
    <b>Proliferative and Hypertrophic</b><br>
    <br>
    This dataset was published by <a href="https://doi.org/10.1371/journal.pbio.2005263" target="_blank">Lui et al., 2018</a> 
    and raw data can be found in the BioProject <a href="https://www.ncbi.nlm.nih.gov/bioproject/PRJNA473132" target="_blank">PRJNA473132</a>.<br>
    Tissue source: Proliferative and Hypertrophic zones from Growth Plate isolated with LCM.<br>
    Age: 1 week old.<br> 
    Sex: Both.<br>
    Layout: PAIRED.<br>
    Reads length: 50 bp.<br>
    Experimental procedure: Freshly frozen bone/cartilage were sectioned and microdissected with laser capture microdissection (LCM) and RNA was extracted with Rneasy Micro Kit
    RNA-Seq libraries were then constructed using Nextera XT DNA Library Preparation Kit (Illumina, San Diego, CA)
    Sequencing was performed with Illumina HiSeq 3000 and normalized to TPM.

</p>
<h3>Bone</h3>
<p>
    Datasets come from two works:<br>
    <br>
    1) This dataset was published by <a href="https://doi.org/10.1038/s41467-019-11373-9" target="_blank">Vrahnas et al. 2019 </a> 
    and raw data can be found in the BioProject <a href="https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA434572" target="_blank">PRJNA434572</a><br>
    Tissue source: Femora.<br>
    Age: 12 weeks old.<br>
    Sex: Females.<br>
    Layout: PAIRED.<br>
    Reads length: 100 bp.<br>
    Experimental procedure: RNA-seq profiles were obtained from femoral bone from 3 ephrin B2 deficient (Dmp1Cre.EfnB2f/f) mice and 4 control mice (Dmp1Cre). 
    Femora were collected, ends removed and marrow flushed out prior to snap freezing in liquid nitrogen. The cortical bone was homogenized in QIAzol lysis reagent using a Polytron PTA 20S homogenised. RNA was extracted using a Rneasy Lipid Tissue Mini kit (Qiagen) according to manufacturer's instructions.
    mRNA reverse transcription cDNA libraries were prepared using the TruSeq RNA Sample preparation kit (Illumina) following the manufacturer’s instructions.
    Sequencing was performed with Illumina HiSeq 2500 and normalized to TPM.<br>
    <br>
    2) This dataset was published by <a href=" https://doi.org/10.1002/jbmr.3819" target="_blank">Davis et al. 2019 </a> 
    and raw data can be found in the BioProject <a href="https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA550395" target="_blank">PRJNA550395</a><br>
    Tissue source: Tibiae.<br>
    Age: 16 weeks old.<br>
    Sex: Males.<br>
    Layout: SINGLE.<br>
    Reads length: 75 bp.<br>
    Experimental procedure: Differential gene expression in Ctrl vs NT3-Cre+ non-loaded and loaded limbs. 16 total samples. n=4 each genotype (Ctrl or NT3-Cre+) as biological replicates. Paired non-loaded and loaded samples for each mouse.
    Right and left tibiae were stripped of all muscle tissue and flushed by centrifugation to remove bone marrow. The flushed tibiae were then cut 2 mm from the articular surface and again at the distal TFJ to isolate the area of peak strain. Tissue samples were flash-frozen in liquid nitrogen and stored at -80C°. Frozen tibiae were pulverized in liquid nitrogen using a Mikro Dismembrator (Mikro-Dismembrator S; B. Braun Biotech International, Melsungen, Germany), resuspended in TRIzol (15596026; Invitrogen, USA), and stored at -80C°. Total RNA was isolated using a Direct-zol kit per manufacturer instructions (R2072; Zymo Research, USA). RNA concentration was determined using a Nanodrop (ND-2000; ThermoFisher, USA). RIN values were used to determine RNA quality (Bioanalyzer 2100; Agilent Technologies, USA).
    RNA sequencing was performed on the Total RNAble platform (Cofactor Genomics, http://cofactorgenomics.com, St. Louis, MO, USA). Libraries were prepared and sequenced as single-end 75 base pair reads on an Illumina NextSeq500 following the manufacturer’s protocols.

</p>
<h3>Tendon</h3>
<p>
    This dataset was published by <a href="https://doi:10.1126/sciadv.abg6069" target="_blank">Cho et al., 2021 </a> 
    and raw data can be found in the BioProject <a href="https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA750927" target="_blank">PRJNA750927</a>.<br>
    Tissue source: Achilles tendon.<br>
    Age: 12 weeks old + 3 or 6 weeks after treatment.<br>  
    Sex: Males.<br>
    Layout: PAIRED.<br>
    Reads length: 100 bp.<br>
    Experimental procedure: The Achilles tendon was separated from soleus muscle and were homogenized using a tissue homogenizer, and total RNA was extracted using TRI reagent and the manufacturer's protocol.
    The sequencing library is prepared by random fragmentation of the DNA or cDNA sample, followed by 5’ and 3’ adapter ligation. Alternatively, “tagmentation” combines the fragmentation and ligation reactions into a single step that greatly increases the efficiency of the library preparation process. Adapter-ligated fragments are then PCR amplified and gel purified.
    Sequencing was performed with Illumina HiSeq 2000 and normalized to TPM.

</p>
<h3>Muscle</h3>
<p>
    This dataset was published by <a href="https://doi.org/10.1016/j.celrep.2023.112336" target="_blank">Mousa et al., 2023</a> 
    and raw data can be found in the BioProject <a href="https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA818005" target="_blank">PRJNA818005</a>.<br>
    Tissue source: Gastrocnemius and soleus muscles.<br>
    Age: 12 weeks old.<br>
    Sex: Males.<br>
    Layout: PAIRED.<br>
    Reads length: 150 bp.<br>
    Experimental procedure: Muscles were harvested and immediately snap frozen in liquid nitrogen. Total RNA was isolated using RNA STAT-60 as per manufacterer's instructions. RNA was DNase I treated as per manufacturer’s instructions (RNase-Free DNase Set, Qiagen) then cleaned up and eluted with RNAse and DNAse free molecular grade water (RNeasy MinElute Cleanup Kit, Qiagen), followed by quantification.
    KAPA RiboErase. Sequencing was performed with Illumina NovaSeq 6000 and normalized to TPM.

</p>