<h2>Single-nucleus of knee tissues from <i>human</i></h2>
<div style="margin-left: 10px;">
    <h3>Tissues</h3>
    <ul>
        <li>Bone</li>
        <li>Cartilage</li>
        <li>Dermis</li>
        <li>Endothelial</li>
        <li>Growth plate</li>
        <li>Interzone</li>
        <li>Lymphatic</li>
        <li>Muscle</li> 
        <li>Schwann</li>
        <li>Tendon</li>
    </ul>

    <!-- <h3>About</h3> -->
    <p>
    This dataset was published by <a href="https://www.nature.com/articles/s41586-024-08189-z" target="_blank">To et al., 2024</a>
    and raw data can be found in ArrayExpress (<a href="http://www.ebi.ac.uk/arrayexpress" target="_blank">www.ebi.ac.uk/arrayexpress</a>)
    under accession number <a href="https://www.ebi.ac.uk/biostudies/arrayexpress/studies/E-MTAB-14385?query=E-MTAB-14385" target="_blank">E-MTAB-14385</a>.
    <br><br>
    <b>Source tissues:</b> Cruciate ligament, femur, meniscus, patella and tibia.<br>
    <b>Age:</b> 5-11 post conception week.<br>
    <b>Sex:</b> Unknown.<br>
    <b>Layout:</b> Paired.<br>
    <b>Reads length:</b> 150 bp.<br><br>
    <b>Experimental procedure:</b> Single-nucleus data were obtained from healthy human embryos of unknown sex, aged between 5 and 11 weeks post-conception (n=10). 
    Cells were isolated from fresh frozen samples through cryosectioning into 10 μm sections, using a homogenization buffer with a Dounce glass mortar (Merck). 
    Tissue dissociation was performed using a loose pestle, followed by additional strokes with a tight pestle if undissociated fragments remained. 
    The mixture was then filtered through a 50 μm filter, centrifuged at 500g for 5 minutes, and the pellet was resuspended in 300 μl of storage buffer. 
    This suspension was filtered again using a 20 μm cell strainer. Nuclear viability was assessed using trypan blue, and samples were processed following the Chromium Single Cell Multiome ATAC + Gene Expression protocol from 10X Genomics. 
    In the nuclear suspensions, 16,000 droplets per reaction were recovered. In some cases, nuclei from different donors were pooled in a single reaction and later distinguished by genotype. 
    Quality control of cDNA and final libraries was performed using High Sensitivity DNA Analysis on a Bioanalyzer (Agilent). 
    Finally, libraries were sequenced on the NovaSeq 6000 system (Illumina), ensuring a minimum sequencing depth of 20,000 read pairs per sample. Data were normalized to CP10K.
    <br><br>
</div>    